Dev-Corner/slides/tool_integration.html

```diff
@@ -1028,15 +1059,54 @@
 
                     <section data-markdown>
                         <script type="text/template">
+                            <img src="../../shared/images/conda_logo.png" class="no-border"  height=50 />
+                            ### Write a Conda package
+
+                            You will have to and you can implement a Conda package if a specific tool or version is not available.
```

Comment:
I would rather say: "You can implement a new Conda package. This will be necessary if a specific tool or version is not available."